src/cmd.rs

```diff
@@ -419,7 +484,49 @@ impl AsRef<str> for PoolOption {
     }
 }
 
-pub fn osd_out(cluster_handle: rados_t, osd_id: u64, simulate: bool) -> Result<(), RadosError> {
+impl fmt::Display for HealthStatus {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        match self {
+            &HealthStatus::HealthErr => write!(f, "HEALTH_ERR"),
```

Comment:
`write!(f, self.as_ref())` ?

Reply:
Compiler says no dice.  Must be a string literal

Reply:
so ... `write!(f, "{}", self.as_ref());` ?